map/set-max-extent/src/main/java/com.esri.samples.set_max_extent/SetMaxExtentLauncher.java

```diff
@@ -0,0 +1,9 @@
+package com.esri.samples.set_max_extent;
+
+/**
+ * Wrapper required for launching a JavaFX 11 app through Gradle or from a jar.
+ */
+public class SetMaxExtentLauncher {
+
+    public static void main(String[] args) { SetMaxExtentSample.main(args); }
```

Comment:
there's no reason why this couldn't be a one liner, but for consistency and better readability, we put this on a new line. Also looks like the IDE indents are out: I'll send a separate message to you about this

```suggestion
  public static void main(String[] args) {
    SetMaxExtentSample.main(args);
  }
```